pyproject.toml

```diff
@@ -52,6 +55,24 @@ ignore = ["D100", "D101", "D102", "D107", "D104", "D213", "D407", "D413"]
 module = "geopandas.*,cerberus.*,osgeo.*,diagrams.*,usaddress.*" # no stubs available
 ignore_missing_imports = true
 
+[tool.coverage.run]
+# Measure branch coverage (https://coverage.readthedocs.io/en/latest/branch.html)
+branch = true
+omit = [
+    "dcpy/test/*",
+]
+
+[tool.coverage.report]
+include_namespace_packages = true
+show_missing = true
+# Regexes for lines to exclude from consideration
+exclude_lines =[
+    # Have to re-enable the standard pragma
+    "pragma: no cover",
+    # Don't complain if non-runnable code isn't run:
+    "if __name__ == .__main__.:",
+]
+
```

Comment:
we were already reporting pytest coverage, these changes are settings I've used before to get a more meaningful/actionable report

[before](https://github.com/NYCPlanning/data-engineering/actions/runs/6397179274/job/17364575811#step:4:22) and [after](https://github.com/NYCPlanning/data-engineering/actions/runs/6397179274/job/17364575811#step:4:22)